portfolio.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+	<link type = "text/css" rel = "stylesheet" href = "wbstylesheet.css"/>
```

Comment:
We should double check the syntax for a link. It's different between portfolio & index.